### PR TITLE
LiveLab 8022 : Lab 4 - Kafka-TEQ code and Load okafka project as library

### PR DESCRIPTION
**Pull request by pasimoes**
This PR has two major components:
- Code for Lab 4: Kafka connection with TEQ
- Okafka loaded as library (local maven repo)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "workshops/eventmesh-teq-kafka/okafka"]
```
Comment (paulparkinson) [inline .gitmodules:1]: why this?

File: .gitignore [hunk 2]
```diff
@@ -501,3 +501,9 @@ workshops/eventmesh-teq-kafka/**/deploy/
 workshops/eventmesh-teq-kafka/cloud-setup/database/
```
Comment (paulparkinson) [inline .gitignore:501]: I think these are, and if not should be, captured at the root level gitignore?

**Review by paulparkinson — COMMENTED** [review]
pretty large PR, I'll review closer once it's in as long as everything is isolated under workshops/eventmesh-teq-kafka which it appears to be. thanks.

File: .gitmodules (hunk 1 above)
Comment (pasimoes) [reply .gitmodules:1]: okafka project is being loaded as submodule (git submodule). I remove the code copied. It is a best practice when we need another project.

File: .gitignore (hunk 2 above)
Comment (pasimoes) [reply .gitignore:501]: yes 👍🏻 
from root .gitignore
## Remove Other Local project stuffs
workshops/eventmesh-teq-kafka/cloud-setup/database/

**Review by paulparkinson — CHANGES_REQUESTED** [review]
There are *a lot* (369) file changes in this PR. Please remove anything that is not self-contained under eventmesh-teq-kafka dir. Mixing in changes from different workshops, even if tested (which I don't think these are), is asking for trouble. Thanks.

**Pull request merged**